commands.go

```diff
@@ -2010,6 +2011,14 @@ func (c cmdable) ZInterStore(ctx context.Context, destination string, store *ZSt
 	return cmd
 }
 
+func (c cmdable) ZMScore(ctx context.Context, key string, members ...interface{}) *FloatSliceCmd {
```

Comment:
I see other `Z*` commands use `member string` instead of `interface{}`. Should we use `...string` here or I am missing something?